microsoft-edge/index.yml

```diff
@@ -0,0 +1,445 @@
+### YamlMime:Hub
+
+title: Microsoft Edge developer documentation
+summary: Microsoft Edge is the browser built for Windows 10, providing fast, safe, and productive web experiences.  Here are all the developer features and tools available to you in Microsoft Edge and the Chromium engine.
+brand: m365
+metadata:
+  title: Microsoft Edge developer documentation
+  description: Microsoft Edge information for developers including PWAs, DevTools, extensions, webdriver, and more.
```

Comment:
I don't think this is entirely accurate. We also include information for IT admins and regular browser users as well.

Reply:
Also, "WebDriver" should be capitalized.

Reply:
Updated.
"Microsoft Edge information for developers, IT admins, and browser end users; including DevTools, Extensions, PWAs, WebDriver, and more."